### DISCUSSION TITLE
Using the default markdown-only template without its header

### DISCUSSION THREAD

**Discussion by DannyBen** [community/community#21758] [Other Feature Feedback, Questions, & Ideas]
<p>Hello everybody,</p>
<p>I am trying to publish a simple one page markdown to GitHub pages.</p>
<p>I would like to use the default “No Theme” theme, so that it only shows the markdown exactly as it is inside GitHub.</p>
<p>So far, I have two outstanding issues, as shown in this screen</p>
<p><img src="https://user-images.githubusercontent.com/2405099/181066567-9e197073-2ff1-4e63-8e17-26a4091b88ec.png" alt="err.png" data-base62-sha1="8f4uonrfYr2HmmMAHrxnlAf7FL8" width="562" height="305"></p>
<ol>
<li>
<p>I need to get rid of the default header that is added</p>
</li>
<li>
<p>HTMLs are not properly rendered, as they do on GitHub itself.</p>
</li>
</ol>
<p>I have tried all sorts of things, like:</p>
<ul>
<li>
<p>adding <code>.nojekyll</code> file</p>
</li>
<li>
<p>adding front matter, setting <code>layout: false</code></p>
</li>
</ul>
<p>I am not sure if I need to create my own layout and move the HTML portion of this markdown to it, and if so, where is the baseline jekyll layout that will look exactly like GitHub markdown.</p>
<p>Thank you in advance for any tips.</p>
<p>EDIT: Solved by </p>
<p><a href="https://github.community/t/remove-heading/10323">https://github.community/t5/GitHub-Pages/Remove-Heading/td-p/4220</a></p>
<p>Sorry, this is a duplicate post…</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hey <a class="mention" href="/dannyben">@dannyben</a>,</p>
<p>Thanks a lot for coming back and sharing your answer! This will perhaps help someone later on :slight_smile:</p>

**Comment by DannyBen** [comment]
<p>Yeah, I hope it does.</p>
<p>I needed it to create my little <a href="https://github.dannyben.com/" rel="nofollow noopener">GitHub Showcase site</a> :slight_smile:</p>
